src/OrchardCore.Modules/OrchardCore.Sms.Azure/Services/AzureSmsProvider.cs

```diff
@@ -0,0 +1,22 @@
+using Microsoft.Extensions.Localization;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using OrchardCore.Sms.Azure.Models;
+
+namespace OrchardCore.Sms.Azure.Services;
+
+public sealed class AzureSmsProvider : AzureSmsProviderBase
+{
+    public const string TechnicalName = "Azure";
+
+    public AzureSmsProvider(
+        IOptions<AzureSmsOptions> options,
+        IPhoneFormatValidator phoneFormatValidator,
+        ILogger<AzureSmsProvider> logger,
+        IStringLocalizer<AzureSmsProvider> stringLocalizer)
+        : base(options.Value, phoneFormatValidator, logger, stringLocalizer)
+    {
+    }
+
+    public override LocalizedString Name => S["Azure Communication Service"];
```

Comment:
```suggestion
    public override LocalizedString Name 
        => S["Azure Communication Service"];
```

Reply:
It could be inline because there's a space

Reply:
I think it is always better to start a new line so this way you don't think about it when you code it. "Is there enough space or not"? The rule I follow, always place it on a new line which will make me not think about it. I just do it.